src/pyspiffe/bundle/jwt_bundle/jwt_bundle.py

```diff
@@ -11,29 +21,37 @@ class JwtBundle(object):
     JwtBundle is a collection of trusted JWT public keys for a trust domain
     """
 
-    def __init__(self) -> None:
-        """Creates an instance of JwtBundle.
-        TODO: complete
+    def __init__(
+        self, trust_domain: TrustDomain, jwt_authorities: Dict[str, _PUBLIC_KEY_TYPES]
+    ) -> None:
+        """Creates a JwtBundle instance.
+
+        Args:
+            trust_domain: The TrustDomain to associate with the JwtBundle instance.
+            jwt_authorities: A dictionay with key_id->PublicKey valid for the given TrustDomain.
         """
-        pass
-        # self.jwt_authorities = {}
+        self._trust_domain = trust_domain
+        if jwt_authorities:
+            self._jwt_authorities = dict(jwt_authorities)
```

Comment:
`jwt_authorities` argument is already of type `Dict`. Is an special reason to use `dict(jwt_authorities)` here?
On a side note, from what I can tell based on some research in the past, `{}` were preferible instead of `dict()` because it is faster.

Reply:
@glaucimar, it would be probably better to use `jwt_authorities.copy()` instead of `dict(jwt_authorities)` to make it clear that you want to copy the collection passed as argument.

Reply:
Hi! dict(jwt_authorities) makes a shallow copy of jwt_authorities (as .dopy() does) but I've read that dict() is faster. This is why I am using dict()

Reply:
I run some benchmarks, `.copy()` looks faster than `dict()`.

Results:

```
Python: 3.8.3

Using dict.copy() 

dict(size=0).copy() x 1000000 times:     0.0710
dict(size=1).copy() x 1000000 times:     0.0897
dict(size=10).copy() x 1000000 times:    0.1083
dict(size=20).copy() x 1000000 times:    0.2184
dict(size=50).copy() x 1000000 times:    0.2687
dict(size=100).copy() x 1000000 times:   0.3999
dict(size=500).copy() x 1000000 times:   2.1426
dict(size=1000).copy() x 1000000 times:  4.6997


Using dict(): 

dict(size=0) x 1000000 times:    0.1757
dict(size=1) x 1000000 times:    0.1936
dict(size=10) x 1000000 times:   0.3684
dict(size=20) x 1000000 times:   0.6323
dict(size=50) x 1000000 times:   1.1137
dict(size=100) x 1000000 times:  2.1038
dict(size=500) x 1000000 times:  10.9229
dict(size=1000) x 1000000 times:   21.6762


Python: 3.6.12

Using dict.copy() 

dict(size=0).copy() x 1000000 times:     0.0983
dict(size=1).copy() x 1000000 times:     0.1128
dict(size=10).copy() x 1000000 times:    0.2716
dict(size=20).copy() x 1000000 times:    0.5310
dict(size=50).copy() x 1000000 times:    0.9969
dict(size=100).copy() x 1000000 times:   1.8560
dict(size=500).copy() x 1000000 times:   9.3752
dict(size=1000).copy() x 1000000 times:  18.8208


Using dict(): 

dict(size=0) x 1000000 times:    0.1694
dict(size=1) x 1000000 times:    0.1924
dict(size=10) x 1000000 times:   0.3473
dict(size=20) x 1000000 times:   0.6102
dict(size=50) x 1000000 times:   1.1310
dict(size=100) x 1000000 times:  1.9679
dict(size=500) x 1000000 times:  10.1202
dict(size=1000) x 1000000 times:   19.2478
```

If you want to try it yourself, I leave you here the benchmark script:

```
def main():
    import time

    N = 1000000

    print('\n\n')
    print('Using dict.copy() \n')

    for size in [0, 1, 10, 20, 50, 100, 500, 1000]:
        d = dict([(str(i), i) for i in range(size)])

        t = time.monotonic()
        for i in range(N):
            d.copy()
        e = time.monotonic() - t

        print(f'dict(size={size}).copy() x {N} times:\t {e:.4f}')

    print('\n\n')
    print('Using dict(): \n')

    for size in [0, 1, 10, 20, 50, 100, 500, 1000]:
        d = dict([(str(i), i) for i in range(size)])

        t = time.monotonic()
        for i in range(N):
            dict(d)
        e = time.monotonic() - t

        print(f'dict(size={size}) x {N} times:\t {e:.4f}')


if __name__ == "__main__":
    main()
```

Reply:
Thanks for this, Max! I will make the change.